e2e/tests/notebook/notebook-download.spec.ts

```diff
@@ -68,6 +68,6 @@ describe('Jupyter notebook download test', () => {
     // As of 9/25/20 I was unable to find a clear mechanism for accessing this.
 
     await notebook.deleteNotebook(notebookName);
-  }, 20 * 60 * 1000);
+  }, 30 * 60 * 1000);
```

Comment:
30 minutes? This is min * sec * millis riight?

Reply:
yes, it's 30 minutes.